imod/mf6/package.py

```diff
@@ -278,6 +279,17 @@ def _validate(self, schemata: Dict, **kwargs) -> Dict[str, List[ValidationError]
                         errors[variable].append(e)
         return errors
 
+    def is_empty(self) -> bool:
+        """
+        Returns True if the package is empty- for example if it contains only no-data values.
+        """
+        allnodata_schemata = filter_schemata_dict(
+            self._write_schemata, (AllNoDataSchema, EmptyIndexesSchema)
+        )
+
+        allnodata_errors = self._validate(allnodata_schemata)
```

Comment:
This might be useful to add:

```python
        # Find if packages throws ValidationError for AllNoDataSchema or
        # EmptyIndexesSchema.
```

Reply:
done

Reply:
done